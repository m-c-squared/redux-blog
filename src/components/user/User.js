import React from 'react';
import PropTypes from 'prop-types';
const User = ({ user }) => {  
  return (
    <>
    <li>
      {user.name}
    </li>
    </>
  );
};

User.propTypes =  {
  user: PropTypes.object.isRequired
};
export default User;  
