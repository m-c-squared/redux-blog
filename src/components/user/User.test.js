import React from 'react';
import renderer from 'react-test-renderer';
import User from './User';
import { MemoryRouter } from 'react-router-dom';

describe('User', () => {
  it('matches a snapshot', () => {
    const user = {
      name: 'Chelsey Dietrich'
    };
    const tree = renderer.create(
      <MemoryRouter>
        <User user={user}/>
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
