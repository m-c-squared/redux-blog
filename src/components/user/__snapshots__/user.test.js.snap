// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User matches a snapshot 1`] = `
<li>
  Chelsey Dietrich
</li>
`;
